tests/parallel_for/hierarchical.cpp

```diff
@@ -33,30 +33,31 @@ int main() {
   const int size = 10;
   std::vector<int> data(size);
   const int groupsize = 2;
-/* Put &data[0] instead of data.data() because it is not obvious in the
-   excerpt below it is a vector */
-//////// Start slide
-buffer<int> input(&data[0], size);
-buffer<int> output(size);
-my_queue.submit([&](handler &cgh)
-{
-  auto in_access = input.get_access<access::mode::read>(cgh);
-  auto out_access = output.get_access<access::mode::write>(cgh);
-
-  cgh.parallel_for_work_group<class hierarchical>(nd_range<>(range<>(size),
-                                                             range<>(groupsize)),
-                                                  [=](group<> group)
+  /* Put &data[0] instead of data.data() because it is not obvious in the
```

Comment:
Please do not change the weird indent in the middle between "Start Slide" and "End Slide" because it is a copy/paste from/to a slide.
Furthermore it is difficult to see what you actually changed to the code.